### PR TITLE
clicking on the sides of main panel and below side panels

### PR DESCRIPTION
**Pull request by qndel**
Fixes vanilla bug that allows casting through quest panel
Allows casting spells on the side of main panel and below side panels
Fixes a bug that didn't allow clicking on the left side of the screen when quest/char panel were open
Center speedbook horizontally and move it above main panel

### CODE REVIEW

File: defs.h [hunk 1]
```diff
@@ -91,8 +91,8 @@
 #define PAL16_RED		224
 #define PAL16_GRAY		240
 
-#define SCREEN_WIDTH	640
-#define SCREEN_HEIGHT	480
+#define SCREEN_WIDTH	1280
```
Comment (AJenbo) [inline defs.h]: Lol you just can't not commit this can you :D

Comment (qndel) [reply defs.h]: XD

Comment (qndel) [reply defs.h]: jesus, how do you handle that? I fail to remember about restoring it every single time lol

Comment (AJenbo) [reply defs.h]: I commit chunks visually in a GIT GUI (git-cola), do you just hit a bit red button in your IDE?

Comment (qndel) [reply defs.h]: I use commands in bash :D

Comment (AJenbo) [reply defs.h]: https://desktop.github.com/

https://www.gitkraken.com/

**Comment by AJenbo** [comment]
This make the speed book not work with a game controller, any chance you could look at that also?

**Pull request merged**